### PR TITLE
el: Update translation

### PR DESCRIPTION
**Pull request by anna-d**
@dspinellis please review

### CODE REVIEW

File: README-el.md [hunk 1]
```diff
@@ -340,9 +349,9 @@ mkdir empty && rsync -r --delete empty/ some-dir && rmdir some-dir
 
 - Είναι εξαιρετικά βοηθητικό ορισμένες φορές το γεγονός ότι μπορείτε να εφαρμόσετε τις πράξεις της τομής, της ένωσης και της διαφοράς σε αρχεία κειμένου με τις εντολές `sort`/`uniq`. Ας πούμε ότι έχετε `a` και `b` αρχεία κείμενου που καμία γραμμή τους δεν επαναλαμβάνεται. Αυτό είναι γρήγορο και δουλεύει σε αρχεία οποιουδήποτε μεγέθους έως πολλά gigabytes. (H `sort` δεν περιορίζεται από τη μνήμη, αν και ίσως χρειαστεί να χρησιμοποιήσετε την επιλογή `-T`, αν ο `/tmp` βρίσκεται σε μικρή κατάτμηση (partition). Επίσης, δείτε την παρατήρηση παραπάνω για τη μεταβλητή `LC_ALL` και την επιλογή `-u` της εντολής `sort` (παραλείπεται παρακάτω ώστε να είναι πιο ξεκάθαρο το παράδειγμα).
 ```sh
-      cat a b | sort | uniq > c   # το c είναι a ένωση b
-      cat a b | sort | uniq -d > c   # c είναι a τομή b
-      cat a b b | sort | uniq -u > c   # το c είναι a διαφορά b
+      sort a b | uniq > c   # το c είναι a ένωση b
+      sort a b | uniq -d > c   # c είναι a τομή b
+      sort a b b | uniq -u > c  # το c είναι η διαφορά του a από το b
```
Comment (dspinellis) [inline README-el.md]: Consider adopting the text of https://github.com/jlevy/the-art-of-command-line/pull/553/commits/1843c1eddc431b570e810fbc59a9c085bdb9176e

**Comment by dspinellis** [comment]
So nice to see this continuously maintained; thank you! I recommend the translation to all my students. I added a few minor comments. No follow-up review is required.

File: README-el.md [hunk 2]
```diff
@@ -105,7 +105,7 @@
 
 - Χρησιμοποιήστε `nohup` ή `disown`, αν θέλετε μία διεργασία παρασκηνίου να συνεχίσει να εκτελείται για πάντα.
 
-- Ελέγξτε ποιες διεργασίες είναι σε αναμονή για κλήση listen με `netstat -lntp` ή `ss -plat` (για TCP· προσθέστε `-u` για UDP).
+- Ελέγξτε ποιες διεργασίες είναι σε αναμονή για κλήση listen με `netstat -lntp` ή `ss -plat` (για TCP· προσθέστε `-u` για UDP) ή `lsof -iTCP -sTCP:LISTEN -P -n` (που δουλεύει επίσης σε OS X).
```
[the remark this answers is not in the document]
Comment (anna-d) [reply README-el.md]: I will update the translation as soon as pull request #552 has been merged.

**Comment by anna-d** [comment]
@dspinellis thank you
@jlevy ready to be merged

**Comment by jlevy** [comment]
👍 wonderful! Thanks so much  @anna-d  and @dspinellis !

**Pull request merged**